.github/workflows/fe-combine-dependabot-prs.yml

```
@@ -104,19 +105,41 @@ jobs:
           git config pull.rebase false
           git config user.name github-actions
           git config user.email github-actions@github.com
-
+          
+          echo $sourcebranches
+          
           git branch $COMBINE_BRANCH_NAME $basebranch
           git checkout $COMBINE_BRANCH_NAME
-          git pull origin $sourcebranches --no-edit
+          
+          EXCLUDED_BRANCHES=''
+                    
+          for sourceBranch in $sourcebranches
+          do
+            {
+              git pull origin $sourceBranch --no-edit
+            } || {
+              git merge --abort
+              echo "Failed branch is ${sourceBranch}"
+              EXCLUDED_BRANCHES+="${sourceBranch} "
+            }
+          done
+          
           git push origin $COMBINE_BRANCH_NAME
+          echo "::set-output name=excluded-branches::$EXCLUDED_BRANCHES"
```

It seems that github will soon [deprecate ](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) `set-output` command.  You should change it to something like that
```suggestion
          echo "excluded-branches=$EXCLUDED_BRANCHES" >> $GITHUB_OUTPUT
```